packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -9,6 +9,7 @@ import * as yargsParser from 'yargs-parser';
 import { showNxWarning, unparse } from './shared';
 import { output } from './output';
 import * as ora from 'ora';
+import * as validateNpmPackage from 'validate-npm-package-name';
```

Comment:
Could just inline the util into the package I think. It hasn't changed in 5 years and is pretty small.

https://raw.githubusercontent.com/npm/validate-npm-package-name/main/index.js

I'd ignore the whole `builtins` thing since no one would be installing a Node builtin as Nx plugin. :)

Reply:
done!